packages/devtools_app/lib/src/logging/logging_controller.dart

```diff
@@ -354,6 +353,7 @@ class LoggingController
         summary: '${changedInfo.extension}: ${changedInfo.value}',
       ));
     } else if (e.extensionKind == 'Flutter.Error') {
+      print('logging error event in loggign controller');
```

Comment:
Should we use a logger here?

Also, sp: `loggign`

Reply:
accidentally included. removed.